GitUI/CommandsDialogs/FormResolveConflicts.cs

```diff
@@ -53,6 +53,8 @@ public partial class FormResolveConflicts : GitModuleForm
         private readonly TranslationString _keepModifiedButtonText = new TranslationString("Keep modified");
         private readonly TranslationString _keepBaseButtonText = new TranslationString("Keep base file");
 
+        private readonly TranslationString _applyToAllCheckBoxText = new TranslationString("Apply to all");
```

Comment:
It would be a better UX to say "this action applies to 2/3/10/... files" as opposed to "all" files. Will it apply to _all_ files or _all selected_ files?
Also the checkbox shouldn't be shown if there is only one file is selected.

Reply:
Yes, you are completely right.